cfgov/jinja2/v1/youth_employment_success/line-item.html

```diff
@@ -2,7 +2,9 @@
 <div class="content-l m-yes-line-item u-clearfix">
   <p class="content-l_col content-l_col-2-3 line-item line-item__content-l h4">
     {{ values.label | safe }}
-    {{ values.helperText | safe if values.helperText else '' }}
+    <span class="line-item__content-l helper">
```

Comment:
`helper` should probably be something less generic, or at least `u-helper`.

Reply:
I actually don't need this class, removing! 😬